README.md

```diff
@@ -37,19 +37,16 @@ icu = "1.5.0"
 
 ```rust
 use icu::calendar::DateTime;
-use icu::datetime::{options::length, DateTimeFormatter};
+use icu::datetime::{NeoFormatter, NeoSkeletonLength, neo_marker::NeoAutoDateTimeMarker};
 use icu::locale::locale;
 
-let options =
-    length::Bag::from_date_time_style(length::Date::Long, length::Time::Medium).into();
-
-let dtf = DateTimeFormatter::try_new(&locale!("es").into(), options)
+let dtf = NeoFormatter::<NeoAutoDateTimeMarker>::try_new(&locale!("es").into(), NeoSkeletonLength::Long.into())
     .expect("locale should be present in compiled data");
 
 let date = DateTime::try_new_iso_datetime(2020, 9, 12, 12, 35, 0).expect("datetime should be valid");
 let date = date.to_any();
 
-let formatted_date = dtf.format_to_string(&date).expect("formatting should succeed");
+let formatted_date = dtf.convert_and_format(&date).to_string_lossy();
```

Comment:
the `expect` that was here before fails?

Reply:
It doesn't fail; the error is on the TryWriteable impl now, thus the `to_string_lossy`

Reply:
This PR creates `to_string_lossy`, an associated function on FormattedNeoDateTime, because I had a lot of docs tests I had to migrate which wanted to use `assert_eq` and otherwise I had to do a convoluted TryWriteable call site. I would like to land this and debate the API later.